test/distributed/test_c10d_nccl.py

```diff
@@ -1524,6 +1524,18 @@ def test_non_blocking_with_eager_init(self):
         self.assertEqual(backend.comm_split_count(), 1)
 
 
+    @requires_nccl()
+    @skip_but_pass_in_sandcastle_if(not TEST_MULTIGPU, "NCCL test requires 2+ GPUs")
+    def test_get_uid(self):
+        store = c10d.FileStore(self.file_name, self.world_size)
+        device = torch.device(f'cuda:{self.rank}')
+        pg = self._create_process_group_nccl(store, self.opts(), device_id=device)
```

Comment:
does this test need to delete all existing pgs, if they were previously created by other unit tests?

Reply:
I think the testcase class has already helped clean pgs and each test will use a new tmp filestore.